hindi/bollywood.md

```diff
@@ -16,4 +16,6 @@ Amithab, [go here](actor3/amithab.md)
 दीपिका पादुकोन  के लिए
 [ यहाँ  ](./actor5/deepika.md)
 
-Art of Living [Know More](art-of-living/art-of-living.md)
\ No newline at end of file
+Art of Living [Know More](art-of-living/art-of-living.md)
+
+[Sunny Deol](sunny_deol/sunny.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.